components/prism-pug.js

```diff
@@ -130,7 +130,9 @@
 						}
 					}
 				],
-				'punctuation': /:/
+				'punctuation': /:/,
+				'attr-id': /#[\w\-]+/,
```

Comment:
One last thing: I think we should name these tokens like the corresponding CSS selector [tokens](https://github.com/PrismJS/prism/blob/c93244768ec7031caa6359655289c1e756a90194/components/prism-css-extras.js#L11), so just `id` and `class`. This should make it easier for someone to write a theme.